internal/scaffold/go_mod.go

```diff
@@ -68,6 +68,8 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.0.0-20191016113814-3b1a734dba6e
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20191016112829-06bb3c9d77c9
 )
+
+replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
 `
```

Comment:
The same here. Why add the mod in a no helm project?

Reply:
See https://github.com/operator-framework/operator-sdk/pull/2080#discussion_r351479087